libs/eo/reports/data-access-api/eo-reports.service.ts

```diff
@@ -41,7 +41,7 @@ export class EoReportsService implements OnDestroy {
   readonly loading = computed(() => this.#loading());
   readonly error = computed(() => this.#error());
 
-  private readonly POLLING_INTERVAL = 5000; // 5 seconds
+  private readonly POLLING_INTERVAL = 5000; // 10 seconds
```

Comment:
isnt it 10000 for 10 sec?

Reply:
We decided to alter it to 5 seconds, I have updated the comment :)